includes/gateway/class-omise-payment-billpayment-tesco.php

```diff
@@ -104,6 +104,8 @@ public function result( $order_id, $order, $charge ) {
 			}
 
 			if ( 'pending' == $charge['status'] ) {
+				$order->update_status( 'on-hold', __( 'Omise: Awaiting Bill Payment to be paid.', 'omise' ) );
+
 				return array(
 					'result'   => 'success',
```

Comment:
the same here @guzzilar, we don't have any const for this result, right?

Reply:
This one also no. They don't provide the constant of `success` result.
Same as https://github.com/omise/omise-woocommerce/pull/142#discussion_r340977243